docs/usage/configuration.md

```diff
@@ -78,8 +78,8 @@ http:
 
 The following additional resources are available, although it is recommended to serve them on a separate listener, not exposed to the public internet:
 
- - `name: prometheus`: serves the a Prometheus-compatible metrics endpoint on `/metrics`, if the Prometheus exporter is enabled in `telemetry.metrics.exporter`.
```

Comment:
s/serves the a Prometheus-compatible/serves a Prometheus-compatible

(remove 'the')